Listenbrainz/Model/Listens/ListensAdditionalInfo.swift

```diff
@@ -5,47 +5,15 @@
 //  Created by Gaurav Bhardwaj on 06/06/23.
 //
 
-//import Foundation
-//
-//// MARK: - ListensAdditionalInfo
-//struct ListensAdditionalInfo: Codable {
```

Comment:
@gauravbhardwaj7 why has this been committed?

Reply:
To get originUrl for opening in spotify from listens

Reply:
I meant the commented code. Remove it

Reply:
commented code has been removed

Reply:
My bad!